frontend/src/components/Form/Toast.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+
+interface iToast {
+  type: string;
+  isActive?: boolean;
+}
+const Toast: FC<iToast> = ({ type, isActive, children }) => {
+  return (
+    <Container type={type} isVisible={isActive}>
+      {children}
+    </Container>
+  );
+};
+
+const Container = styled.section<{ isVisible?: boolean; type: string }>`
```

Comment:
`type` can be removed from here as you're not doing anything with it with styled components

Reply:
if i remove it TS starts complaining